src/Input.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export default function Input(
+  { informations: { name, category, address }, onChange, handleClick },
```

Comment:
```suggestion
  {
     informations: { name, category, address },
     onChange, handleClick
   },
```

이정도로 길어지면 줄 바꿈을 해주는 게 좋습니다.

information의 의미를 알기 어렵네요. 레스토랑에 관한 내용이니

restaurant은 어떨까요?

Reply:
그렇네요 ㅋㅋ restaurant이 좋은거 같아요